eslint.config.mjs

```diff
@@ -0,0 +1,164 @@
+import ts from '@typescript-eslint/eslint-plugin'
+import tsParser from '@typescript-eslint/parser'
+import prettier from 'eslint-config-prettier'
+import react from 'eslint-plugin-react'
+import reactHooks from 'eslint-plugin-react-hooks'
+import testingLibrary from 'eslint-plugin-testing-library'
+import jest from 'eslint-plugin-jest'
+import globals from 'globals'
+
+export default [
+  'eslint:recommended',
+  // Ignored dirs
+  {
+    ignores: ['dist/**/*', 'compiled/**/*', 'bundle/**/*']
+  },
+  // Temporary ignored dirs - TODO: remove on rewrite
+  {
+    ignores: ['main/signers/**/*']
+  },
+  // All files
+  {
+    files: ['**/*.{js,mjs,ts,tsx}'],
+    languageOptions: {
+      ecmaVersion: 'latest',
+      globals: {
+        ...globals.es2021
+      }
+    },
+    rules: {
+      'no-unused-vars': [
+        'error',
+        {
+          args: 'after-used',
+          ignoreRestSiblings: true,
+          argsIgnorePattern: '^_',
+          destructuredArrayIgnorePattern: '^_'
+        }
+      ]
+    }
+  },
+  // Main process files and scripts
+  {
+    files: ['**/*.{js,mjs,ts}'],
+    ignores: ['app/**/*', 'resources/Components/**/*'],
+    languageOptions: {
+      globals: {
+        ...globals.node
+      }
+    }
+  },
+  // Renderer process files
+  {
+    files: [
+      'app/**/*.js',
+      'main/dapps/server/inject/*.js',
+      'resources/**/*.{js,ts,tsx}',
```

Comment:
then here we can just specify the only directory we care about here, `resources/Components`